.github/workflows/ci.yml

```
@@ -368,6 +368,61 @@ jobs:
           name: ${{ matrix.image_name }}-docker-image
           path: "${{ matrix.image_name }}-docker-image"
 
+  build-and-push-internal-docker:
+    timeout-minutes: 5
+    needs: [build-setup, build]
+
+    name: Build and Push Internal Docker Image
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        image_name: ${{ fromJson(needs.build-setup.outputs.image_names) }}
+
+    permissions:
+      contents: "read"
+      id-token: "write"
+
+    env:
+      PLATFORMS: "${{ join(fromJson(needs.build-setup.outputs.platforms), ',') }}"
+      # This name is made up we might want to go with a different one
+      INTERNAL_DOCKER_IMAGE: "us-central1-docker.pkg.dev/internal/relay/${{ matrix.image_name }}"
```

```suggestion
      INTERNAL_DOCKER_IMAGE: "us-central1-docker.pkg.dev/internal-sentry/relay/${{ matrix.image_name }}"
```

Would be the correct path.